invenio_rdm_records/ext.py

```diff
@@ -22,8 +22,9 @@
     RDMRecordFilesResourceConfig, RDMRecordResource, RDMRecordResourceConfig
 from .secret_links import LinkNeed, SecretLink
 from .services import RDMFileDraftServiceConfig, RDMFileRecordServiceConfig, \
-    RDMRecordService, RDMRecordServiceConfig
+    RDMRecordService, RDMRecordServiceConfig, SecretLinkServiceConfig
 from .services.schemas.metadata_extensions import MetadataExtensions
+from .services.secret_links.service import SecretLinkService
```

Comment:
When refactoring one of the main points is to try not to have to change imports. This is to avoid "breaking compatibilty". In this case you can avoid having this import (and leaving it as it was) by "exporting" the class in the `__init__.py`, note that you can do that successively (until you export it in `.services.__init__.py`).